app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -50,6 +51,17 @@ open class FirefoxApplication : LocaleAwareApplication() {
             // Enable crash reporting. Don't add anything above here because if it crashes, we won't know.
             SentryIntegration.init(this, serviceLocator.settingsRepo)
 
+            // Set up Glean by connecting it to the preference flag for data collection and calling
+            // the initialize() function.
```

Comment:
Nit: It seems like comment mostly describes existing code, so I think we should be safe to remove it.

I do see a review comment above saying that `setUploadEnabled` needs to be set before `initialize`, and that seems like value context.  I think we should comment about that.